### PR TITLE
Searchable orgs

### PR DESCRIPTION
**Pull request by drewvolz**
This adds the searchbar and searching capabilities to Student Orgs for iOS.

### CODE REVIEW

File: source/views/components/html-view.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-import {WebView, StyleSheet} from 'react-native'
+import {WebView} from 'react-native'
 import openUrl, {canOpenUrl} from '../components/open-url'
 
 export class HtmlView extends React.Component {
   props: {
     html: string,
     baseUrl?: ?string,
-    style?: ?string,
+    style?: Object,
```
Comment (hawkrives) [inline source/views/components/html-view.js]: `number | Object | Array<number | Object>` pleases 😁

File: source/views/student-orgs/detail.ios.js [hunk 2]
```diff
@@ -2,7 +2,7 @@
 import React from 'react'
 import {ScrollView, Text, StyleSheet, Linking} from 'react-native'
 import moment from 'moment'
-import {HtmlView} from '../components/html-view' 
+import {HtmlView} from '../components/html-view'
```
Comment (hawkrives) [inline source/views/student-orgs/detail.ios.js:5]: Thanks!

File: source/views/student-orgs/list.js [hunk 3]
```diff
@@ -58,11 +68,13 @@ const styles = StyleSheet.create({
 export class StudentOrgsView extends React.Component {
   state: {
     orgs: {[key: string]: StudentOrgType[]},
+    results: {[key: string]: Array<StudentOrgType>},
```
Comment (hawkrives) [inline source/views/student-orgs/list.js]: `StudentOrgType[]`, please, for consistency with the line above.

File: source/views/student-orgs/list.js [hunk 4]
```diff
@@ -173,6 +184,43 @@ export class StudentOrgsView extends React.Component {
     })
   }
 
+  splitToArray = (str: string) => {
```
Comment (hawkrives) [inline source/views/student-orgs/list.js:187]: At some point (not this PR!) we should really pull this out into its own function.

And, like, make some documentation of those functions.

**Review by hawkrives — CHANGES_REQUESTED** [review]
Wow! Thanks!

File: source/views/student-orgs/list.js (hunk 4 above)
Comment (drewvolz) [reply source/views/student-orgs/list.js:187]: Agreed.

**Comment by drewvolz** [comment]
I would consider this PR as acceptable unless there are further changes anyone would like made.

**Review by hawkrives — APPROVED** [review]
Sure, let's merge it into my branch.

**Pull request merged**